webknossos/webknossos/cli/convert_knossos.py

```diff
@@ -246,15 +251,21 @@ def main(
         typer.Option(help="Name of the cubed layer (color or segmentation)"),
     ] = "color",
     voxel_size: Annotated[
-        VoxelSize,
+        VoxelSizeTuple,
         typer.Option(
             help="The size of one voxel in source data in nanometers. "
             "Should be a comma separated string (e.g. 11.0,11.0,20.0).",
             parser=parse_voxel_size,
-            metavar="VOXEL_SIZE",
+            metavar="VoxelSize",
             show_default=False,
         ),
     ],
+    length_unit: Annotated[
+        LengthUnit,
+        typer.Option(
+            help="The unit of the voxel size.",
+        ),
+    ] = "nanometer",  # type:ignore
```

Comment:
that’s a lot of "nanometer" strings in all these files. Maybe a DEFAULT_UNIT (and maybe additionally DEFAULT_UNIT_STR) or something like that could be defined somewhere, next to the VoxelSize class, and be imported here?

Reply:
Done.